tests/php/bootstrap.php

```diff
@@ -42,13 +42,16 @@ function amp_unit_test_load_plugin_file() {
 
 tests_add_filter( 'muplugins_loaded', 'amp_unit_test_load_plugin_file' );
 
-// Start up the WP testing environment.
-require $_test_root . '/includes/bootstrap.php';
-
 /*
  * Load WP CLI. Its test bootstrap file can't be required as it will load
  * duplicate class names which are already in use.
  */
 define( 'WP_CLI_ROOT', TESTS_PLUGIN_DIR . '/vendor/wp-cli/wp-cli' );
 define( 'WP_CLI_VENDOR_DIR', TESTS_PLUGIN_DIR . '/vendor' );
 require_once WP_CLI_ROOT . '/php/utils.php';
+
+$logger = new WP_CLI\Loggers\Regular( true );
+WP_CLI::set_logger( $logger );
```

Comment:
What is the warning that is being emitted?

Reply:
The `amp-block-editor.js` file was previously not being emitted by Webpack (which is now fixed in https://github.com/ampproject/amp-wp/pull/6078/commits/4b47698af6363c6f183a71cc559adb3e0a2baf07). Because of that the following error was being emitted:

https://github.com/ampproject/amp-wp/blob/1e7ea02cee63910aa5fe20c38e9f588cb6bdb170/amp.php#L160-L169

Reply:
So now the warning is not being emitted? But if it were to be emitted again, then the logger change will ensure the warning will be emitted without a failure?

Reply:
Yes and yes.